web/client/plugins/GeoStory.jsx

```diff
@@ -28,19 +35,49 @@ import MediaContentToolbar from './geostory/MediaContentToolbar';
 const GeoStory = ({
```

Comment:
We need to add documentation for this plugin for the new property as requested in the issue description. If possible It would be nice to have also some tests on this plugin in particular for the new introduced useEffect